### PR TITLE
Create class for Floor

### PR DESCRIPTION
**Pull request by ferAguilar02**
This class will receive tiles and according to those determine how many points will be deducted

### CODE REVIEW

File: src/main/java/jalau/at18/azul/CenterTileBoard.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package jalau.at18.azul;
+import java.util.*;
+
+public class CenterTileBoard extends ArrayList<Character> {
+
+    public CenterTileBoard removeTilesOfSpecificColor(char color) {
```
Comment (J0c4) [inline src/main/java/jalau/at18/azul/CenterTileBoard.java:6]: Wasn't this removed on a previous PR?

**Review by J0c4 — COMMENTED** [review]
Looks good! however, please check my question and confirm if there are things to remove from this PR

File: src/main/java/jalau/at18/azul/CenterTileBoard.java (hunk 1 above)
Comment (ferAguilar02) [reply src/main/java/jalau/at18/azul/CenterTileBoard.java:6]: Yes it was.  I just realized I created this branch from the other one. I will fix it right now

Comment (ferAguilar02) [reply src/main/java/jalau/at18/azul/CenterTileBoard.java]: I have fixed the conflicts that came up

**Review by J0c4 — APPROVED** [review]

**Pull request merged**